pyproject.toml

```diff
@@ -3,14 +3,14 @@ requires = ["setuptools>=61.0", "setuptools-scm"]
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "my-conda-subcommand"
-version = "1.0.0"
+name = "string-art"
+version = "1.0"
 description = "My string art subcommand plugin"
 requires-python = ">=3.7"
-dependencies = ["conda", "art"]
+dependencies = ["conda", "pyfiglet"]
```

Comment:
Could we please also change the [subcommand module](https://github.com/conda/conda-plugin-template/blob/main/string_art.py) to import `pyfiglet` vs `art`?

Reply:
ah I had been planning to do that as a separate PR (this accidentally slipped through), let me revert this to `art` and then we can do that separately